prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -204,6 +204,12 @@ class DetailedSubmissionHistory(
         return destinations.filter { it.itemCount != 0 }.size
     }
 
+    /**
+     * Container for the sum of items in total that have passed any filters
```

Comment:
Found this wording a little confusing, maybe this instead?
```suggestion
     * Number of items in total that have passed any filters
```